tests/manager/manager-executeWriteCommand_error-002.phpt

```diff
@@ -21,13 +21,73 @@ try {
     $manager->executeWriteCommand(DATABASE_NAME, $command, ['writeConcern' => new MongoDB\Driver\WriteConcern("undefined")]);
 } catch (MongoDB\Driver\Exception\CommandException $e) {
     printf("%s(%d): %s\n", get_class($e), $e->getCode(), $e->getMessage());
-    assert($e->getMessage() === "Write Concern error: " . $e->getResultDocument()->writeConcernError->errmsg);
-    assert($e->getCode() === $e->getResultDocument()->writeConcernError->code);
+    var_dump($e->getResultDocument());
 }
 
 ?>
 ===DONE===
 <?php exit(0); ?>
---EXPECT--
+--EXPECTF--
 MongoDB\Driver\Exception\CommandException(79): Write Concern error: No write concern mode named 'undefined' found in replica set configuration
+object(stdClass)#13 (6) {
+  ["lastErrorObject"]=>
+  object(stdClass)#5 (3) {
+    ["n"]=>
+    int(1)
+    ["updatedExisting"]=>
+    bool(false)
+    ["upserted"]=>
+    string(3) "foo"
+  }
+  ["value"]=>
+  object(stdClass)#6 (2) {
+    ["_id"]=>
+    string(3) "foo"
+    ["foo"]=>
+    array(1) {
+      [0]=>
+      string(3) "bar"
+    }
+  }
+  ["writeConcernError"]=>
+  object(stdClass)#7 (3) {
+    ["code"]=>
+    int(79)
+    ["codeName"]=>
+    string(23) "UnknownReplWriteConcern"
+    ["errmsg"]=>
+    string(74) "No write concern mode named 'undefined' found in replica set configuration"
+  }
+  ["ok"]=>
+  float(1)
+  ["operationTime"]=>
```

Comment:
Since `operationTime` and `$clusterTime` are 3.6+ fields, let's replace these with a wildcard pattern as is done in some other tests.

```
  ["ok"]=>
  float(1)%A
}
```